src/platform/silabs/efr32/BUILD.gn

```diff
@@ -19,6 +19,14 @@ import("${chip_root}/src/crypto/crypto.gni")
 import("${chip_root}/src/platform/device.gni")
 import("${chip_root}/third_party/silabs/silabs_board.gni")
 
+declare_args() {
+  #default WiFi SSID
+  chip_default_wifi_ssid = ""
```

Comment:
When would this ever be used for a non-testing-example app?  This seems like an odd thing to do in platform...